src/opm/parser/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp

```diff
@@ -395,6 +432,17 @@ inline void keywordAquifer( SummaryConfig::keyword_list& list,
     }
 }
 
+
+std::string completion_name(const std::string& keyword, int completion) {
+    if (completion < 10)
+        return fmt::format("{}__{}", keyword, completion);
+
+    if (completion < 100)
+        return fmt::format("{}_{}", keyword, completion);
+
+    return fmt::format("{}{}", keyword, completion);
```

Comment:
Doesn't `fmt::format()`  already do the requisite padding/filling?
```C++
#include <fmt/format.h>

int main()
{
    fmt::print("{}{:_>3}\n", "COPRL",   5);
    fmt::print("{}{:_>3}\n", "COPRL",  15);
    fmt::print("{}{:_>3}\n", "COPRL", 123);
}
// Output:
// COPRL__5
// COPRL_15
// COPRL123
```

Reply:
Cool - I did not know; will update.